src/third-party/pinyin/pinyin_data.hpp

```diff
@@ -0,0 +1,454 @@
+    {U"zhi", U"㕄㗌㗧㘉㙷㛿㜱㜼㝂㡳㡶㣥㥀㧗㨁㨖㩼㫑㮹㯄㲍㲛㲳㴛㴯㸟㽻㿃䃽䄺䅩䆈䇛䇧䇽䉅䉜䌤䎺䏄䏯䐈䐭䑇䓋䓌䓜䓡䕌䖨䘣䘭䚦䚳䛊䛗䝰䝷䞃䞠䟈䟡䡹䣽䤠䥍䦯䧝䧴䩢䬹䭁䱃䱥䲀䳅䵂䵹之乿伎侄俧倁値值偫傂儨凪制剬劕劧卮厎厔只吱呮咥咫嗭嚔址坁坧垁埃埴執墆墌夂妷姪娡媞嬂寘實峙崻巵帋帙帜幟庢庤廌彘徏徔徝徵志忮怾恃恉慹憄懥懫戠执扺扻抧抵拓挃指挚捗掷搘搱摕摨摭摯擲擳擿支斦旘旨昵晊晢智杝杫枝枳柣栀栉栺桎梔梽植椥楖榰樀樲樴櫍櫛止歭殖氏氐汁汥汦沚治泜洔洷淔淽滍滞滯漐潌潪瀄炙熫犆狾猘璏瓆瓡畤疐疷疻痔痣瘈直眰知砋砥礩示祁祇祉祑祗祬禃禔秇秓秖秩秪积秲秷稙稚稺穉窒筫紙紩絷絺綕緻縶織纸织置翐耆聀职職肢胑胝胵脂膣膱至致臷臸芖芝芷茋茝菭薙藢蘵虒蚔蛭蜘螲蟙衹衼袟袠製襧覟觗觝觯觶訨誌識豑豒豸貭質贄质贽趾跂跖跱踬踯踶蹛蹠蹢躑躓軄軹軽輊轵轾迣遟遲郅酈酯釞鉄銍銴鋕鑕铚锧阤阯陁陟隲隻雉馶馽駤騭騺驇骘鯯鳩鳷鴙鴲鶨鷙鸷黹鼅𠊤𠊷𠋤𠍜𠓶𠘖𠚅𠟅𠦧𠮡𠰅𠼠𡀹𡁉𡂒𡂣𡈊𡈼𡌴𡍶𡏀𡏚𡑘𡖧𡖻𡙑𡠗𡠹𡮞𡰹𡱔𡸜𡽆𢃜𢄢𢄱𢅁𢇨𢊁𢍧𢎈𢐂𢕞𢖇𢖿𢙺𢚨𢛍𢡒𢦕𢧤𢯶𢰙𢴠𢴧𢴨𢷸𢻙𢽃𢽗𢾫𣔐𣖌𣖭𣖿𣗻𣚠𣥰𣨋𣱐𣲵𣳀𣽚𤃲𤆒𤓕𤖞𤛱𤞂𤞌𤦄𤦮𤧜𤴛𤴟𤴢𤵋𤶓𤸓𤽁𤿙𥃫𥇕𥇭𥍭𥎹𥏄𥏅𥏊𥏰𥏷𥒓𥒗𥔊𥘡𥙆𥝑𥝮𥠈𥠽𥣮𥭡𥮖𥴒𥹩𥻬𥾣𥿮𦃘𦏤𦐖𦛧𦜋𦝔𦟔𦤻𦥎𦥏𦥐𦭜𦭮𦯫𦯯𦰘𦳮𦴀𧀿𧃐𧊙𧌔𧏸𧐉𧓸𧙁𧛢𧜚𧝉𧠫𧠴𧣭𧣾𧤡𧨰𧩼𧫡𧱒𧸅𧸕𧸲𧹛𧼴𧽦𧾂𨁷𨂂𨃯𨆧𨈙𨌌𨎉𨎌𨑨𨒉𨕕𨖋𨖹𨙸𨜎𨟊𨟾𨡐𨢮𨤱𨧵𨫔𨬚𨰛𨵂𨻆𨼓𨿛𩅀𩊝𩊴𩋩𩍲𩍵𩙾𩧄𩬺𩯈𩷓𩹈𩻼𪁊𪁓𪁩𪂅𪉆𪏀𪐏𪑜𪒊𪗨𪗻𪙹𫛛𬃊"},
+    {U"zhong", U"㐺㣫㲁㲴㳊㹣㼿䇗䈺䝦䱰中乑仲伀众偅冢刣喠堹塚塜夂妐妕媑尰幒彸徸忠忪柊橦歱汷泈潼炂煄狆瘇盅眾祌种種穜童筗籦終緟终肿腫舂舯茽董蔠蚛蚣蝩螤螽蟲衆衳衶衷褈諥踵蹱重鈆鈡銿鍾鐘钟锺鴤鼨𠊥𠛀𠱧𡖌𡥿𡰒𡻑𢁷𢃭𢝆𢨱𣱧𣷡𣹞𤚏𤝅𤯚𤺄𥗦𥷈𥻝𦉂𦌋𦔉𦬕𧆼𧑆𧬤𧳮𨉢𨳗𩅞𩅧𩾋𩿀𬑔"},
+    {U"zhou", U"㑇㑳㗙㛩㣙㤘㥮㨄㫶㼙㾭㿒䈙䋓䎇䎻䏔䐢䑼䓟䖞䛆䧓䩜䶇伷侏侜倜僽冑周呪咒咮啁啄喌喙噣嚋妯婤宙州帚徟扭掫昼晝晭柚椆注洀洲淍炿烐珘甃疛皱皺盩睭矪碡祝箒籀籒籕粙粥紂紬縐繇纣绉翢肘育胄胕舟舳荮菷葤薵詋詶調諏謅譸诌诪賙赒軸輈輖轴辀逐週郮酎鈾銂霌駎駲騆騶驟骤鬻鯞鵃鸼𠊣𠋢𠚴𠣘𠤍𠱙𡀑𡊡𢃸𢏝𢐫𢓟𢫧𢷗𢼲𢽧𣆔𣥯𣻱𤏲𤵥𥀙𥌆𥑸𥖠𥣙𥲝𥺝𥺞𥼫𥾓𥿦𦁖𦂈𦅸𦈺𦡴𦩈𦭴𧇟𧛸𧣷𧧔𧭍𧳜𧻖𨉜𨏺𨥇𨦞𨨯𩊄𩋰𩌄𩍌𩍧𩗪𩢸𩧨𩧳𩶣𪆀𪇞"},
+    {U"zhu", U"㑏㑳㔉㝉㤖㦵㧣㫂㵭㶆㹥㺛㾻㿾䃴䆝䇠䇡䇬䌵䍆䎷䐗䐢䕽䘄䘚䘢䝒䝬䟉䠱䡤䣷䥮䪒䬡䭖䮱䰞䳠丶主之予伫佇住侏兪劚助劯咮嗻嘱噣囑坾墸壴孎宁宔尌屬嵀庶拄搊敱斀斗斸曯朝朮术朱杼枓柠柱柷株楮槠樦橥櫡櫧櫫欘殶泏泞注洙渚潴澍濐瀦灟炢炷烛煑煮燭爥猪珠疰瘃眝瞩矚砫硃磩祝祩秼窋竚竹竺笁笜筑筯箸築篫篴紵紸絑纻罜羜翥舳芧苎苧茁茱茿莇著蓫薥藷藸蚰蛀蛛蝫蠋蠩蠾袾褚註詝誅諸诛诸豬貯贮跓跙跦躅軴軸迬逐逗逫邾鉏鉒銖鋳鑄钃铢铸阻除陼霔飳馵駐駯騶驻鮢鯺鱁鴸鸀麆麈鼄𠧀𠩈𠮌𠰍𠴦𠴫𠷅𡎺𡤗𡧨𡱱𡴅𡸌𡺐𡻌𡻠𢁼𢔪𢚻𢥃𢩄𢲿𣀻𣔯𣤁𣥼𣵸𣽆𤆼𤋰𤎧𤕞𤝹𤥮𤲑𤳯𤾄𥋛𥛂𥞏𥩣𥯸𥵟𥹍𥾅𦅷𦉐𦙴𦧙𦬸𧈚𧉞𧏿𧑤𧵤𨆄𨈫𨙔𨞕𨥇𨭅𨲈𩊣𩋵𩒊𩞈𩣷𩨻𩲠𩲬𩳥𩴀𩶂𩶄𪊹𪋏𪋑𪋰𪏿𪚹𬣞"},
+    {U"zhua", U"抓摣撾檛爪簻膼髽𠆣𡎬𣑃𥬲𥮣"},
+    {U"zhuai", U"尵拽睉跩顡𢶀"},
+    {U"zhuan", U"㛯㼷䉵䏙䏝䡱䧘专傳僎僝剸叀啭囀堟塼嫥孨専專巽恮摶撰沌湍漙灷瑑瑼甎砖磗磚竱篆篹篿簨籑縳耑腞膞蒃蟤襈諯譔賺赚転轉转鄟顓颛饌馔鱄𠊩𠨎𡇰𡢀𡤛𡩚𡭇𡰞𢂘𢐎𢞬𣂵𣕏𣚢𤂤𤩄𤪪𤮳𥛥𥫛𦁆𦄯𦓝𦝏𦠆𦧸𧂍𧶲𧸖𨷱𩔊𩧜𩳏𩻝𬤥"},
+    {U"zhuang", U"䚒僮壮壯壵奘妆妝娤幢庄庒憧戇撞桩梉樁湷漴焋状狀獞粧糚艟荘莊装裝贛𠌴𢙳𢤤𢦅𣞝𣴣𣶍𣻛𤕯𤘲𤶜𦀜𦚏𩅃𩮱𩯲𪁈𪄻𪅖𪉉"},
+    {U"zhui", U"㗓㚝㩾㮅㾽䄌䢡䧳䨨䳡䶆倕坠垂墜娷惴揣桘椎槌沝甀畷硾磓礈笍箠綴縋缀缒腏膇致萑諈贅赘轛追醀醊錐錗錣鎚鑆锥隊隧隹餟騅骓鵻𠄌𡑻𡯑𢊅𣝸𣦬𣨫𤺅𥟒𦥻𧿲𨒉𨧨𨪗𨺵𨾻𩛵𩜀𩪀𩬳𪋇"},
+    {U"zhun", U"㡒䐃准凖啍圫埻宒屯忳敦旽淳準甽盹稕窀純綧肫胗衠訰諄谆踆迍飩𥇜𥚠𬘯"},
+    {U"zhuo", U"㑁㒂㓸㣿㧳㧻㪬㭬㹿㺟䂐䃗䅵䆯䐁䓬䕴䖦䞵䟾䦃䪼䫎䮓䮕䵠䵵䶂丵倬剢劅勺卓叕啄啅啜噣圴墌妰娺彴拙捉捔掇撯擆擢斀斫斮斱斲斵晫杓桌梲棁棳棹椓槕櫡汋浊浞涿淖準濁濯灂灼炪烵焯燋犳狵琢琸着矠硺禚穛穱窡窧箸篧籗籱繳罬聉肫茁著蓔蕞藋蝃蠗蠿諁諑謶诼趠趵踔蹠躅酌鉵鋜鐯鐲镯鵫鷟𠡑𠭴𠿡𡷿𢁁𢢗𢧈𢳇𢺡𢽚𣃈𣃑𣄻𤃮𤉐𤏸𤓦𥇍𥋮𥐊𥗁𥞺𥢔𥮥𥯩𥷘𥷮𥼚𦜰𦰹𦳡𧂒𧃔𧘑𧞐𧢼𧨳𧰵𧱒𧱰𨁿𨑽𨖮𨡸𨢬𨧧𨮿𨺝𩆸𩋁𩑂𩩔𩲃𩷹𬸦"},
+    {U"zi", U"㜽㞨㠿㧗㧘㬐㰣㰷㱴㺭㽧㾅㿳䁒䅆䅔䆅䎩䐉䔂䖪䘣䣎䦻䰵乲事仔倳兹剚吇吱呰呲咨啙嗞姉姊姕姿子孖字孜孳孶崰嵫恣杍柴栥梓椔榟橴次沝泚洓淄渍湽滋滓漬澬牸玆璾甾疵眥眦矷禌秄秭秶稵穧笫籽粢紎純紫緇缁耔胏胔胾自芓茈茊茡茲荢菑葘蓻薋虸觜訾訿諮谘貲資赀资赼趑趦跐載輜輺辎鄑釨鈭鋅錙鍿鎡锱镃頾頿髭鯔鰦鲻鶅鼒齊齍齜龇𠀢𠂔𠡸𡉗𡗈𡙛𡥎𡪒𡸟𡸪𢆶𢱆𢼱𣄮𣓊𣖨𣚀𣚁𣣊𣣌𣥨𣯃𣳩𣸆𤲸𥀖𥕁𥚉𥞎𥫞𥬳𥲕𥻍𥼩𥼻𥿩𦍺𦎸𦖺𦞤𦣹𦺱𧀗𧂐𧆰𧕓𧙁𧛏𧣤𧥕𧧕𧨴𧹌𧿞𨀥𨍢𨚖𨝳𨧫𨩲𨹀𩄚𩐍𩜊𪅵𪑿𪕊𪗉𪗋𪗐"},
+    {U"zong", U"㙡㚇㢔㣭㨑㯶㷓㹅䁓䈦䍟䑸䖲䗥䙕䝋䡯䢨䰌倊倧偬傯堫宗嵏嵕嵸從总惣惾愡捴揔搃摠昮朡棕椶樅潈潨熜熧燪猔猣疭瘲碂磫稯粽糉糭綜緃総緵縂縦縱總纵综翪腙艐葼蓗蓯蝬豵踨踪蹤錝鍐鏓鑁騌騣骔鬃鬉鬷鯮鯼𠏭𠕌𠡻𠵻𡕰𡞧𣀒𣯨𤡆𥍺𥓻𥚾𥠡𦖸𦡙𧺣𨌰𨍈𨍉𨎢𨺡𩋯𩤗𩦲𩭤𩮀𩰽𪖁"},
+    {U"zou", U"㔌㔿㵵㻓䅳䠫偢奏媰掫揍搊族棷棸楱箃緅芻菆諏诹走赱趣邹郰鄒鄹陬騶驺鯐鯫鲰黀齱齺𠂑𣙻𣠏𥋜𥶈𧌗𨂡𨃘𨑿𨜗𨝮𨽁𩼦𪃆"},
+    {U"zu", U"㞺㧻㩆㰵㲞㵀䃚䅸䔃䖕䙘䚝䯿䱣伹俎倅傶卆卒哫唨啐嘁姐岨崒崪怚族柤槭沮淬爼珇砠祖租稡箤組綷组苴菹葅蒩詛謯诅趲足踤踿蹴鉃鉏鉐錊鎐鎺鏃鑿镞阻靻顇駔𠻏𡻬𢅪𢉺𢫵𢳇𢳈𣇙𣢰𣤶𣨛𤓵𤬧𤱌𤽱𥛜𥞯𥣆𥼀𥼪𦑋𦵬𧇈𧇿𧎲𧐈𧑙𧗎𧞰𧺒𨂀𨃭𨄕𨧰𨨳𨩰𩐡𩥿𩩠𩲲𩺯𪋍𪘧𪙳"},
+    {U"zuan", U"㸇䂎䌣䡽䤸䰖劗揝撮攢攥欑篹籫繤纂纉纘缵賺躜躦鑚鑽钻𡉺𣀶𣪁𦆈𦙉𨇃𨉖𨰭𩎑𩛻𩯳"},
+    {U"zui", U"㝡㠑㭰㰎䓱䘒䘹䮔䯿厜嗺嘴噿嫢嶉嶊嶵摧撮晬最朘枠栬槜槯樶檇檌欈濢璻睟祽稡穝絊纗罪羧蕝蕞蟕觜辠酔酨酻醉鋷錊雋𠲋𠾋𠿘𡙭𡡔𡽁𡽛𢈡𢊛𣖱𣩑𥍋𥳣𥳵𦆈𦈬𦏳𦙈𦸺𧎹𧻝𨢅𨻵𨿇𩚻𩣷𩲨𪋌𪓌"},
+    {U"zun", U"䔿僎僔噂墫壿奠尊嶟拵捘捽撙栫樽瀳繜罇袸譐跧踆蹲遵銌鐏鱒鳟鶎鷷𠟃𠱜𤮐𥊭𥞘𥢎𦢐𦨆𦪚𨱔𩯄"},
+    {U"zuo", U"㑅㘀㘴㘸㛗㝾㤰㩇㭮㵶㸲䋏䎰䔘䝫䞢䞰䟄䟭䟶䦈乍佐作侳做咗唑嘬坐岝岞左座怍挫捽撮昨柞柮椊琢砟祚秨稓笮筰糳繓胙苲莋葃葄蓙袏諎迮酢醋鈼鑿阼飵𠂇𠱯𠹠𡪳𡯨𡹥𢀡𢂃𢤎𣠹𣩈𣹧𤿀𥅁𥙀𥥏𥹁𥽿𦁎𦈛𦥬𦦹𧃘𧨀𧮙𧲭𨀨𨐳𨝨𨞒𩛠𪎇𪎲"},
+    {U",", U"，"},
+    {U".", U"。"},
+    {U"?", U"？"},
+    {U"!", U"！"},
+    {U"-", U"—"},
+    {U"\"", U"“”"}
```

Comment:
```suggestion
    {U"\"", U"“”"},
    {U"(", U"（"},
    {U")", U"）"}
```
Full-width brackets are often used in Chinese item names.